Mathlib/Algebra/Ring/Defs.lean

```diff
@@ -47,7 +47,7 @@ assert_not_exists mul_rotate
 
 universe u v w x
 
-variable {α : Type u} {β : Type v} {γ : Type w} {R : Type x}
+variable {α : Type u} {R : Type x}
```

Comment:
Are v and w now unused universes? Can you rename the universe containing R to v then?

Reply:
Universes renamed!